test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js

```diff
@@ -142,7 +148,72 @@ describe('email notifications settings', () => {
     })
   })
 
-  describe('edit refund email settings', () => {
+  describe('Refund email settings', () => {
+    describe('When email collection mode is MANDATORY or OPTIONAL', () => {
+      beforeEach(() => {
+        cy.task('setupStubs', [
+          gatewayAccountStubs.setRefundEmailEnabledByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, 'false')
+        ])
+      })
+
+      it('should navigate to the refund email toggle page', () => {
+        ['MANDATORY', 'OPTIONAL'].forEach(mode => {
+          setupStubs(ADMIN_ROLE, mode)
+          cy.visit(EMAIL_NOTIFICATIONS_URL)
+          cy.get('.govuk-summary-list').within(() => {
+            cy.get('.govuk-summary-list__actions a').eq(2).click()
+          })
+
+          cy.title().should('contains', 'Settings - Email notifications')
+          cy.url().should('include', '/settings/email-notifications/refund-email-toggle')
+          cy.get('.govuk-fieldset__heading').first().should('contain', 'Do you want to send refund emails?')
 
+          cy.get('.govuk-radios').within(() => {
+            cy.get('.govuk-radios__item').eq(0).should('contain', 'Yes')
+            cy.get('.govuk-radios__item').eq(1).should('contain', 'No')
+          })
+
+          cy.get('input[type="radio"][value="false"]').check()
+          cy.get('.govuk-button').contains('Save changes').click()
+          cy.get('h1').should('contain', 'Email notifications')
+          cy.title().should('eq', 'Settings - Email notifications - GOV.UK Pay')
```

Comment:
We don't need to check persistence, because that would involve a fuller integration test with a database to store a data change, which isn't what we want to do for frontend (not our pay-frontend, but projects for frontend pages) tests.